Tools/scripts/generate_global_objects.py

```diff
@@ -284,6 +285,13 @@ def generate_runtime_init(identifiers, strings):
                 printer.write('')
                 with printer.block('.tuple_empty =', ','):
                     printer.write('.ob_base = _PyVarObject_IMMORTAL_INIT(&PyTuple_Type, 0)')
+        printer.write('')
+        printer.write("static inline void")
+        with printer.block("_PyUnicode_InitStaticStrings(void)"):
+            printer.write(f'PyObject *string;')
+            for i in sorted(identifiers):
+                printer.write(f'string = &_Py_ID({i});')
```

Comment:
As noted above, the earlier comment might be more helpful here.  Also, with a slight adjustment:

```suggestion
                # This use of _Py_ID() is ignored by iter_global_strings()
                # since iter_files() ignores .h files.
                printer.write(f'string = &_Py_ID({i});')
```

Reply:
done